### DISCUSSION TITLE
Map generation using LIDAR data

### DISCUSSION THREAD

**Discussion by GyDomonkos** [OpenOrienteering/mapper#2233] [Ideas]
Since OCAD gives its users the possibility to generate background maps and form lines out of LIDAR data and import them into the map. Currently, I think this is the only major advantage of OCAD compared to OOM. 
Since there already exist programs (like rusty pullauta and Lastools) that do the major part of it, it would make sense to incorporate them into OOM.

I hope I've made my idea clear enough. If I would know the code better, I would try doing this enhancement myself, but I lack the necessary knowledge on how the OOM code is built up.

**Comment by dg0yt** [comment]
FTR there are #915 and https://github.com/OpenOrienteering/mapper/discussions/2061.
